src/antlr/Parser.g

```diff
@@ -980,7 +980,7 @@ alterTableStatement returns [AlterTableStatement.Raw stmt]
       | K_ALTER ( K_IF K_EXISTS { $stmt.ifColumnExists(true); } )? id=cident
               ( mask=columnMask { $stmt.mask(id, mask); }
               | K_DROP K_MASKED { $stmt.mask(id, null); }
-              | K_DROP K_CHECK { $stmt.dropConstraints(id); }
+              | K_DROP K_CHECK { $stmt.alterConstraints(id, null); }
               | (constraints=columnConstraints) { $stmt.alterConstraints(id, constraints); })
```

Comment:
It isn't new with this patch but to be consistent with other similar methods (e.g. `mask/add/drop/rename`), I would suggest renaming `alterConstraints` to just `constrain`

Reply:
What do you think about having its own Jira ticket for it?

Reply:
I don't think it's worth a JIRA to change the name of an internal method. If you don't disagree with the suggestion, why not just change it here?